src/main/java/org/folio/bulkops/service/BulkOperationService.java

```diff
@@ -174,9 +175,13 @@ public BulkOperation uploadCsvFile(EntityType entityType, IdentifierType identif
           operation.setProcessedNumOfRecords(numOfLines);
           operation.setMatchedNumOfRecords(numOfLines);
 
-        } catch (Exception e) {
-          log.error(ERROR_STARTING_BULK_OPERATION + e.getCause());
+        } catch (S3ClientException e) {
           errorMessage = ERROR_NOT_UPLOAD_FILE_S3_INVALID_CONFIGURATION;
```

Comment:
Taking into account this error message will be available in dev console and used for troubleshooting, it makes sense to add information about exception cause here.